bin/pycbc_make_faithsim

```diff
@@ -176,7 +176,7 @@ if __name__ == "__main__":
             bname = "bank/bank" + btag + ".xml"
             if bname not in btables:
                 indoc = utils.load_filename(bname, False, contenthandler=LIGOLWContentHandler)
-                btables[bname] = table.get_table(indoc, lsctables.SimInspiralTable.tableName) 
+                btables[bname] = lsctables.SimInspiralTable.get_table(indoc)
```

Comment:
This is code-in-a-string, right? Is `lsctables` properly imported?

Reply:
Duh, yes it is.

Reply:
This, and pycbc_make_banksim, need removing and replacing with proper pegasus workflows. (Or in the case of banksim, the new model needs proper documentation).